docs/walkthrough.md

```diff
@@ -138,7 +138,16 @@ chmod +x ./kubectl
 We'll assume hereafter that all `kubectl` commands are using this
 newly-installed executable.
 
-To configure `kubectl` to communicate with the service catalog APU server:
+## Step 5 - Configuring `kubectl` to Talk to the API Server
+
+To configure `kubectl` to communicate with the service catalog API server, we'll have to
+get the IP address of the `Service` (and just the IP - no ports or `http://` prefixes)
+that sits in front of the API server pod(s). If you installed the catalog with one of the 
+`helm install` commands above, then this service will be called `catalog-catalog-apiserver`, 
+and be in the `catalog` namespace.
+
+When you determine the IP address of this service, set its value into the `SVC_CAT_API_SERVER_IP`
+environment variable and then run the following commands:
```

Comment:
do we want to suggest an `export SVC_CAT_API_SERVER_IP=$(minikube ip)` or something? Doesn't have to be minikube related, purely used as an example.